proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/MetsUtils.java

```diff
@@ -884,7 +885,7 @@ public static void saveInfoFile(String path, MetsContext metsContext, String md5
             validation.setValue("W3C-XML");
             validation.setVersion(Float.valueOf("0.0"));
             infoJaxb.setValidation(validation);
-            infoJaxb.setCreator(metsContext.getCreatorOrganization());
+            infoJaxb.setCreator(NdkExportOptions.getNdkExportOptions(null).getCreator());
```

Comment:
getNdkExportOptions(null)??? To určitě ne. MetsContext je asi správné místo, kde ty options držet. Aspoň se zdá, že to tak Robert zamýšlel.